aci_access_policies.tf

```diff
@@ -301,6 +301,62 @@ module "aci_link_level_policy" {
+  description            = try(each.value.description, "")
+  cipher_suite           = try(each.value.cipher_suite, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.cipher_suite)
+  key_server_priority    = try(each.value.key_server_priority, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.key_server_priority)
+  window_size            = try(each.value.window_size, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.window_size)
+  key_expiry_time        = try(each.value.key_expiry_time, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.key_expiry_time) == "disabled" || try(each.value.key_expiry_time, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.key_expiry_time) == 0 ? 0 : each.value.key_expiry_time
+  security_policy        = try(each.value.security_policy, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.security_policy)
+  confidentiality_offset = try(each.value.confidentiality_offset, local.defaults.apic.access_policies.interface_policies.macsec_parameters_policies.confidentiality_offset)
+}
+
+locals {
+  macsec_keychain_policies = flatten([
+    for mkc in try(local.access_policies.interface_policies.macsec_keychain_policies, []) : {
+      name        = "${mkc.name}${local.defaults.apic.access_policies.interface_policies.macsec_keychain_policies.name_suffix}"
+      description = try(mkc.description, "")
+      key_policies = [for kp in try(mkc.key_policies, []) : {
+        name           = kp.name
+        key_name       = kp.key_name
+        pre_shared_key = kp.pre_shared_key
+        description    = try(kp.description, "")
+        start_time     = try(kp.start_time, local.defaults.apic.access_policies.interface_policies.macsec_keychain_policies.start_time)
+        end_time       = try(kp.end_time, local.defaults.apic.access_policies.interface_policies.macsec_keychain_policies.end_time)
+        }
+      ]
+    }
+  ])
+}
+
+module "aci_macsec_keychain_policies" {
+  source = "./modules/terraform-aci-macsec-keychain-policies"
+
+  for_each     = { for mkc in try(local.macsec_keychain_policies, []) : mkc.name => mkc if local.modules.aci_macsec_keychain_policies && var.manage_access_policies }
+  name         = "${each.value.name}${local.defaults.apic.access_policies.interface_policies.macsec_keychain_policies.name_suffix}"
+  description  = each.value.description
+  key_policies = each.value.key_policies
+}
+
+module "aci_macsec_interfaces_policy" {
+  source = "./modules/terraform-aci-macsec-interfaces-policy"
+
+  for_each                 = { for mip in try(local.access_policies.interface_policies.macsec_interfaces_policy, []) : mip.name => mip if local.modules.aci_macsec_interfaces_policy && var.manage_access_policies }
```

Comment:
should be plural: local.access_policies.interface_policies.macsec_interfaces_policies